config/config_helpers.go

```diff
@@ -314,14 +315,25 @@ func getAWSAccountID(include *IncludeConfig, terragruntOptions *options.Terragru
 
 	identity, err := sts.New(sess).GetCallerIdentity(nil)
 	if err != nil {
-		return "", errors.WithStackTrace(err)
+		return sts.GetCallerIdentityOutput{}, errors.WithStackTrace(err)
 	}
 
-	return *identity.Account, nil
+	return *identity, nil
 }
 
-// Custom error types
+// Return the AWS account id associated to the current set of credentials
+func getAWSAccountID(include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
+	identity, err := getAWSCallerID(include, terragruntOptions)
+	return *identity.Account, err
```

Comment:
I know this is a copy from the old code, but should this actually be updated to `aws.StringValue(identity.Account)` (import is `"github.com/aws/aws-sdk-go/aws"`)?

Reply:
It looks like in the newest sdk it's `"github.com/aws/aws-sdk-go/aws/awsutil"`

Reply:
be30c58d

Reply:
Ah good to know!